communities/asu-phoenix/community-assessment.md

```diff
@@ -0,0 +1,121 @@
+### Definition
+
+- Students at Arizona State University and the Phoenix Area interested in growing and becoming a part of the tech community. 
+
+### Size
+
+- 20000+ students enrolled at Arizona State University 
+    - 5,650 enrolled in Computing Program
+- 200-300 at large scale hackathons    
```

Comment:
Whoa, this is a huge number! Is that students from ASU attending external large scale hackathons, or the size of the hackathons held on campus?